src/en/clouds.md

```diff
@@ -0,0 +1,154 @@
+Title: Juju clouds
+TODO: Needs to explain available auth types for clouds
+  
+# Clouds
+
+Juju ships with the ability to seamlessly use a number of public clouds
+(including Amazon Web Services, Azure, Google Compute Engine, Joyent and
+Rackspace) to deploy workloads. 
+
+The knowledge to run these clouds is 'baked-in', so for the majority of
+use-cases there is no additional configuration to be done - you can simply
+specify the cloud you wish to use, supply Juju with some
+[credentials][credentials] and start deploying services.
+
```

Comment:
Mention the ability to also leverage private clouds like OpenStack. You mention it later (## Specifying additional clouds) but it would be good to say something at this time.